libs/test-utils/src/console.test.ts

```diff
@@ -1,41 +1,31 @@
 /* eslint-disable no-console */
+import { expect, test } from 'vitest';
 import { suppressingConsoleOutput } from './console';
 
 test.each(['log', 'warn', 'error'] as const)(
   'suppressingConsoleOutput replaces console.%s with a mock',
-  (method) => {
-    suppressingConsoleOutput(() => {
+  async (method) => {
+    const original = console[method];
+    await suppressingConsoleOutput(() => {
       console[method]('test');
       expect(console[method]).toHaveBeenNthCalledWith(1, 'test');
     });
-    expect('mock' in console[method]).toEqual(false);
```

Comment:
This may still be true with `vitest`, but felt like testing an implementation detail. Replaced it with what we actually care about.